

PERSON_ID_NAME_VOICE = [
    {'person_id': 17223924946, 'name': 'Jessica Rust', 'voice_id': 1},
    {'person_id': 117227880, 'name': 'Steve Carrlec', 'voice_id': 5},
    {'person_id': 5299113, 'name': 'John Bravo', 'voice_id': 4},
    {'person_id': 6024224946, 'name': 'Margo Stewart', 'voice_id': 2},
    {'person_id': 1886024346, 'name': 'Evie Lester', 'voice_id': 3}
]

NUM_GROUPS = 4

GROUP_PI_ASSIGMENTS = [
    {'group_id': 1, 'pi_ids': [1,2]},
    {'group_id': 2, 'pi_ids': [3,4,5]},
    {'group_id': 3, 'pi_ids': [6]},
    {'group_id': 4, 'pi_ids': [7,8,9,10,11]}
]



# self.pi_person_assignment = [
#     {'group_id': 1, 'members': [
#         {'pi_id': 1, 'person_id': 5},
#         {'pi_id': 2, 'person_id': 6},
#         {'pi_id': 3, 'person_id': 3},
#     ]},
#     {'group_id': 2, 'members': [
#         {'pi_id': 4, 'person_id': 4},
#         {'pi_id': 5, 'person_id': 7}
#     ]},
#     {'group_id': 2, 'members': [
#         {'pi_id': 6, 'person_id': 1}
#     ]},
#     {'group_id': 2, 'members': [
#         {'pi_id': 7, 'person_id': 2},
#         {'pi_id': 8, 'person_id': 8},
#         {'pi_id': 9, 'person_id': 10},
#         {'pi_id': 10, 'person_id': 9},
#         {'pi_id': 11, 'person_id': 11}
#     ]},
# ]
